frontend/lib/scripts/create.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// this is copied from here and modified: https://github.com/facebook/create-react-app/blob/main/packages/babel-preset-react-app/create.js
+
+// Using custom babel plugin. This is needed for preserving es modules for glide data grid
```

Comment:
Is this file only required because of glide data grid?

Reply:
Yep this file is needed specifically because of glide data grid. glide-data-grid has an exports field that point to cjs when it’s imported using `require()` syntax and we need to keep es modules.

Reply:
hmm, ok. Do you know what we would need to change in the glide-data-grid to get this working out of the box?